lib/sprockets/uri_utils.rb

```diff
@@ -137,15 +138,24 @@ def parse_file_digest_uri(uri)
     # Examples
     #
     #   build("/tmp/js/application.js")
-    #   # => "file-digest:/tmp/js/application.js"
+    #   # => "file-digest:///tmp/js/application.js"
```

Comment:
:boom: I like that better. There's a few checks to make sure these are valid "uris' but we can probably get rid of some of those checks.